### ISSUE TITLE
Designations for CommComm members on Chair

### ISSUE THREAD

**Issue by maddhruv** [CodeDocsJECRC/community-committee#6]
Glad to see the community and work growing :smile: 
I think we should now designate CommComm members @CodeDocsJECRC/community-committee to some positions and assign duties.
Again stating **no designation would be hierarchical, neither veto nor supreme powers shall be assigned to anyone**.

Positions I propose - 
- CommComm Chair - Lead CommComm
- Github Chair - Lead Github organization and members participation
- Outreach Chair - Lead Outer participations and engagements
- Web Co-chairs - Lead for Web development and projects
- Android Co-chairs - Lead for Android development and projects

Other positions are welcome

**Comment by ShivSoni5** [comment]
M not in favour of any designation!

**Comment by maddhruv** [comment]
The designations are not hierarchical but are actually duties allocation, who should govern which part and job!
So as that person can be named responsible for the task

**Comment by ShivSoni5** [comment]
According to me, we should not assign the positions to any CommComm members. This can lead to conflicts in future.

**Comment by amanvijayv** [comment]
I think we can go with Dhruv's proposed positions, it will help us to divide the tasks very easily. It will help us to look more official. And members on those positions can guide volunteer under their guidance. Everyone agree??

**Comment by amanvijayv** [comment]
I think we can go with Dhruv's proposed positions, it will help us to divide the tasks very easily. It will help us to look more official. And members on those positions can guide volunteer under their guidance. Everyone agree??

**Comment by maddhruv** [comment]
+1 @amanvijayv

**Comment by akshaysharma2277** [comment]
I agree with @ShivSoni5 and @maddhruv  instead of assigning position, just divide the work between the team members.

**Comment by AmanBansal1998** [comment]
It is up to you , only.

**Comment by AmanBansal1998** [comment]
I will be for github chair .

**Comment by maddhruv** [comment]
> I will be for github chair

i cannot understand for this part
